mycroft/skills/settings.py

```diff
@@ -268,7 +268,7 @@ def _load_settings_meta(self):
                     if json_file:
                         data = json.load(f)
                     else:
-                        data = yaml.load(f)
+                        data = yaml.load(f, Loader=yaml.SafeLoader)
```

Comment:
Minor nit, but I think it would be simpler to use `yaml.safe_load`, rather than pass in the loader.

Reply:
True that, change incoming!

Reply:
Updated